procedures/pulpcore/update.md.erb

```diff
@@ -0,0 +1,17 @@
+# Collect list of Python packages
+
+* [ ] Execute the following script in an empty EL7 container:
+```bash
+yum install -y postgresql-devel gcc python3 python3-devel python3-pip
+pip3 install pulpcore pulp-file pulp-container
+pip3 freeze |sed '/certifi/d'
+```
+* [ ] Save the `pip` output to a file
+
+# Update `foreman-packaging`
+
+* [ ] Use the list of Python packages and update their RPM packages:
+```bash
+while read line; do pkg=$(echo ${line} |cut -f1 -d=); version=$(echo ${line} | cut -f3 -d=); ./add_pypi_package.sh ${pkg} ${version} katello-pulpcore-nightly-el7 pulpcore; done < THE_PIP_LIST_FILE
```

Comment:
this needs https://github.com/theforeman/foreman-packaging/pull/4478 or some other solution to not regenerate existing packages

Reply:
Using bash instead of cut:
```suggestion
while read line; do pkg=${line%==*} ; version=${line#*==} ; ./add_pypi_package.sh ${pkg} ${version} katello-pulpcore-nightly-el7 pulpcore; done < THE_PIP_LIST_FILE
```

The very short version:

```suggestion
while read line; do ./add_pypi_package.sh ${line%==*} ${line#*==} katello-pulpcore-nightly-el7 pulpcore; done < THE_PIP_LIST_FILE
```

Reply:
Yeah, the long version is good :)